sklearn/externals/joblib/_dask.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function, division, absolute_import
 
```

Comment:
please double check that there's no redundant blank line (e.g., blank line at the beginning of a file).